tests/tests/tests.py

```diff
@@ -4,9 +4,13 @@
 """
 import itertools
 
+try:
+    from unittest import skipIf, skipUnless
+except ImportError:  # Python < 2.7
+    from django.utils.unittest import skipIf, skipUnless
```

Comment:
This is your commit e93c3c65517 . I'm not sure. I don't see any problem with importing `unittest`.